frontend/src/components/commons/SystemMessage.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+
+export const MESSAGE_TYPE = {
+  warning: "warning",
+  error: "error"
+};
+
+//only useful for unit test
+export let UNIT_TEST_VALIDATION = {
+  validation: ""
+};
+
+const styles = {
+  logo: {
+    fontSize: "40px"
+  },
+  textBox: {
+    textAlign: "left",
+    paddingLeft: 25
+  }
+};
+
+class WarningMessage extends Component {
```

Comment:
Your class name here should match your file name.

Reply:
oups.. fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/125/commits/64ba3a08bdf8667c15ac0af91bdb4e6a99efbcc8